pkg/application/registryConfigUseCase.go

```diff
@@ -0,0 +1,235 @@
+package application
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/ecosystem"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+//go:embed testdata/doguConfigDiffMockData.yaml
+var doguConfigDiffMockDataBytes []byte
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>var `doguConfigDiffMockDataBytes` is unused